src/app/router/Options/index.tsx

```diff
@@ -9,11 +9,1130 @@ import "~/i18n/i18nConfig";
 import Options from "./Options";
 
 // Make sure to bind modal to your appElement (https://reactcommunity.org/react-modal/accessibility/)
-Modal.setAppElement("#options-root");
+// Modal.setAppElement("#options-root");
 
 // Get the active theme and apply corresponding Tailwind classes to the document
-getTheme();
+// getTheme();
 
-const container = document.getElementById("options-root") as HTMLElement;
-const root = createRoot(container);
-root.render(<Options />);
+// const container = document.getElementById("options-root") as HTMLElement;
+// const root = createRoot(container);
+// root.render(<Options />);
+
+var $jscomp = $jscomp || {};
```

Comment:
inlined legacy.min.js here. At the bottom of the file is the same code as in my codepen here:
https://codepen.io/jankoegel/pen/yLEmGxJ